media/static-bundles.json

```diff
@@ -1635,6 +1640,7 @@
         "js/base/mozilla-client.js",
         "js/base/nav-main-resp.js",
         "js/base/class-list-polyfill.js",
+        "js/base/object-assign-polyfill.js",
```

Comment:
As we're adding a new polyfill to the base bundle for every page, I would recommend pushing this branch to `run-integration-tests`, just to make sure we're not activating any new code paths that have previously not been tested in older browsers.